react/src/base/inputs/HugeSelect.stories.js

```diff
@@ -1,169 +0,0 @@
-import React from 'react';
```

Comment:
Do we need a netlify redirect?

Reply:
old was https://react.sparkdesignsystem.com/?path=/story/components-input-huge-select--huge-select-box

Reply:
new is, `?path=/story/components-input-select--huge-select-box`